src/Admin/TimelineMeasureAdmin.php

```diff
@@ -59,6 +62,27 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                 'label' => 'Titre',
                 'show_filter' => true,
             ])
+            ->add('profiles', null, [
+                'label' => 'Profils',
+                'show_filter' => true,
+            ], null, ['multiple' => true])
+            ->add('themes', null, [
+                'label' => 'Thèmes',
+                'show_filter' => true,
+            ], null, ['multiple' => true])
+            ->add('status', ChoiceFilter::class, [
+                'label' => 'Statut',
+                'show_filter' => true,
+                'field_type' => ChoiceType::class,
+                'field_options' => [
+                    'multiple' => true,
+                    'choices' => Measure::STATUSES,
+                ],
+            ])
+            ->add('major', null, [
+                'label' => 'Mise en avant (32)',
```

Comment:
Why `(32)`? Forgotten debug or specific request?

Reply:
They just want `Mise en avant (32)` this to be the label of this specific field.

Reply:
nice catch though, after update they are consistent in every admin now